apps/src/aiTutor/views/AITutorSuggestedPrompts.tsx

```diff
@@ -149,25 +149,30 @@ const AITutorSuggestedPrompts: React.FunctionComponent = () => {
     [studentCode, isWaitingForChatResponse, level, dispatch]
   );
 
+  // We set selected to false because once the user selects a prompt, we convert
+  // the chip into a message in the chat history.
   const suggestedPrompts = [
     {
       label: QuickActions[ActionType.COMPILATION],
       onClick: () => handleClick(ActionType.COMPILATION),
       show: showCompilationOption,
+      selected: false,
     },
     {
       label: QuickActions[ActionType.VALIDATION],
       onClick: () => handleClick(ActionType.VALIDATION),
       show: showValidationOption,
+      selected: false,
```

Comment:
Catching up on AI tutor-related work - just leaving a link to a comment about this suggested prompt from a prior PR. https://github.com/code-dot-org/code-dot-org/pull/64046/files#r1975991691

Reply:
Sorry I'm flagging this late but just noticed this:
- user runs their code and there is an error
- suggested prompt 'Why doesn't my code work?' is displayed
- user clicks on suggested prompt and assistant message is generated along with the `AssistantMessageFeedback` at the end.
- suggested prompt 'Why doesn't my code work?' is displayed again.
- user clicks on thumbs up/down button and textarea is displayed but suggested prompt is covering up the text area.

Maybe we can prevent the 2nd suggested prompt from being displayed?

https://github.com/user-attachments/assets/82a0125d-08bd-4fda-9b52-ff78b61ebb7e